includes/types/CMB2_Type_Associated_Objects.php

```diff
@@ -120,11 +135,12 @@ public function render( $args = array() ) {
 
 		// Open our retrieved, or found posts, list
 		echo '<div class="retrieved-wrap column-wrap">';
-		echo '<h4 class="associated-objects-section">' . sprintf( __( 'Available %s', 'cmb' ), $this->post_type_labels ) . '</h4>';
+		echo '<h4 class="associated-objects-section">' . sprintf( __( 'Available %s', 'cmb' ), implode('/', $object_type_labels ) ) . '</h4>';
```

Comment:
It's silly, but let's be sure to keep the WP standards (e.g. `implode( '/'...`)

Reply:
Also, it seems that the `implode()` part should maybe happen inside the `get_all_object_type_labels` method?